src/elements/content-sidebar/__tests__/MetadataInstanceEditor.test.tsx

```diff
@@ -28,12 +52,28 @@ describe('MetadataInstanceEditor', () => {
         isBoxAiSuggestionsEnabled: true,
         isUnsavedChangesModalOpen: false,
         template: mockMetadataTemplate,
-        onCancel: jest.fn(),
+        onCancel: mockOnCancel,
         onDelete: jest.fn(),
         onSubmit: jest.fn(),
-        setIsUnsavedChangesModalOpen: jest.fn(),
+        setIsUnsavedChangesModalOpen: mockSetIsUnsavedChangesModalOpen,
+        onUnsavedChangesModalCancel: mockOnUnsavedChangesModalCancel,
     };
 
+    // Mock window.matchMedia to simulate media query behavior for tests
+    // in which UnsavedChangesModal component relies on it.
+    const mockMatchMedia = () =>
+        Object.defineProperty(window, 'matchMedia', {
+            writable: true,
+            value: jest.fn().mockImplementation(query => ({
+                matches: false,
+                media: query,
+                onchange: null,
+                addEventListener: jest.fn(),
+                removeEventListener: jest.fn(),
+                dispatchEvent: jest.fn(),
+            })),
+        });
+
```

Comment:
this was just added to master, in the global jest setup file, https://github.com/box/box-ui-elements/blob/f5f1bac2eb2b750f748fdf1d8ad6901cb2f7042b/scripts/jest/jest-setup.ts#L5 rebase and remove this and verify if you still need this